deeplink/build.gradle.kts

```diff
@@ -10,37 +16,96 @@ android {
         minSdkVersion(Android.minSdk)
         targetSdkVersion(Android.sdk)
         versionName = Project.version
-        setProperty("archivesBaseName", Project.name)
+        setProperty("archivesBaseName", "${Project.name}-${Project.version}")
         testInstrumentationRunner = "android.support.test.runner.AndroidJUnitRunner"
     }
 }
 
 group = Project.groupId
 version = Project.version
 
-val artifact = "$buildDir/outputs/aar/${Project.name}-${Project.version}.aar"
+dependencies {
+    implementation(Dependencies.kotlinStdLib)
+    implementation(Dependencies.okio)
+    testImplementation(DependenciesTest.junit)
+    testImplementation(DependenciesTest.kotlinTest)
+    androidTestImplementation(DependenciesTest.junit)
+    androidTestImplementation(DependenciesTest.kotlinTest)
+    androidTestImplementation(DependenciesTest.supportTestRunner)
+}
+
+fun MavenPom.addDependencies() = withXml {
```

Comment:
Let's move this extension to the bottom of the file to avoid clutter with the other configs